usaspending_api/common/serializers.py

```diff
@@ -78,6 +77,38 @@ def __init__(self, *args, **kwargs):
                     # We don't have get default fields available
                     pass
 
+    '''
```

Comment:
Tiny thing--can you move this great info into the docstring of `setup_eager_loading`?